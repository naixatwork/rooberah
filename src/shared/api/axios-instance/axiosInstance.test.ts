import { describe, it, expect } from 'vitest';
import axiosInstance from './axiosInstance';

describe('axiosInstance', () => {
  it('should match the snapshot', () => {
    expect(axiosInstance.defaults).toMatchSnapshot();
  });
});
