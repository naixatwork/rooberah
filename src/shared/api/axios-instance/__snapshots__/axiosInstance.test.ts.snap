// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`axiosInstance > should match the snapshot 1`] = `
{
  "adapter": [
    "xhr",
    "http",
    "fetch",
  ],
  "baseURL": "https://dummyjson.com/",
  "env": {
    "Blob": [Function],
    "FormData": [Function],
  },
  "headers": {
    "common": {
      "Accept": "application/json, text/plain, */*",
      "Content-Type": undefined,
    },
    "delete": {},
    "get": {},
    "head": {},
    "patch": {},
    "post": {},
    "put": {},
  },
  "maxBodyLength": -1,
  "maxContentLength": -1,
  "timeout": 0,
  "transformRequest": [
    [Function],
  ],
  "transformResponse": [
    [Function],
  ],
  "transitional": {
    "clarifyTimeoutError": false,
    "forcedJSONParsing": true,
    "silentJSONParsing": true,
  },
  "validateStatus": [Function],
  "xsrfCookieName": "XSRF-TOKEN",
  "xsrfHeaderName": "X-XSRF-TOKEN",
}
`;
